sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/StreamingSqlTest.java

```diff
@@ -39,16 +39,15 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** StreamingSQLTest. */
+/** StreamingSqlTest. */
```

Comment:
I know it isn't new, but this comment should be more descriptive. (Maybe `Tests for ZetaSQL windowing functions (TUMBLE, HOP, and SESSION).`)

Reply:
Done.